pkg/kubectl/cmd/describe.go

```diff
@@ -102,6 +102,7 @@ func NewCmdDescribe(parent string, f cmdutil.Factory, streams genericclioptions.
 
 	cmd := &cobra.Command{
 		Use: "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)",
+		Aliases: []string{"desc"},
```

Comment:
@SataQiu thanks for your first contribution to kubernetes!

This change lgtm, but it first needs to be confirmed by someone from sig-cli if this is ok to add.
/cc @soltysh @apelisse @seans3

Reply:
Currently we don't use aliases for subcommands. We use aliases for resources instead.

Reply:
I agree with what was said before. I don't want to introduce additional layer of confusion as we refactor the code. So no aliases (for now).